features/gebruik_in_lopende_tekst.feature

```diff
@@ -1,6 +1,6 @@
 # language: nl
 
-@proxy @post-assert
+
 Functionaliteit: Als gemeente wil ik de juiste en consistent naamgebruik in een lopende tekst
```

Comment:
--> `Functionaliteit: Als gemeente wil ik een consistent en juist naamgebruik in een lopende tekst